BookPlayer/Settings/Storage/StorageRowView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  StorageRowView.swift
+//  BookPlayer
+//
+//  Created by Dmitrij Hojkolov on 28.06.2023.
+//  Copyright © 2023 Tortuga Power. All rights reserved.
+//
+
+import SwiftUI
+import BookPlayerKit
+
+struct StorageRowView: View {
+  let item: StorageItem
+  let onDeleteTap: (() -> Void)?
+  let onWarningTap: (() -> Void)?
+
+  @EnvironmentObject var themeViewModel: ThemeViewModel
+
+  var body: some View {
+    HStack {
+      Button {
+        onDeleteTap?()
+      } label: {
+        Image(systemName: "minus.circle.fill")
+          .resizable()
+          .scaledToFit()
+          .frame(width: 22, height: 22)
+          .foregroundColor(.red)
+      }
+      .padding(15)
+
+      VStack(alignment: .leading, spacing: 2) {
+        Text(item.title)
+          .font(Font(Fonts.title))
+          .multilineTextAlignment(.leading)
+          .foregroundColor(themeViewModel.primaryColor)
+
+        Text(item.path)
+          .font(.subheadline)
```

Comment:
I think this `.subheadline` and the next one, needs to be a size smaller, to emulate what we have on the library cells, that the title is bigger, and the rest of labels a bit smaller